docs/articles_en/openvino-workflow/running-inference/inference-devices-and-modes/npu-device.rst

```diff
@@ -131,17 +141,13 @@ offer a limited set of supported OpenVINO features.
 Limitations
 #############################
 
-* Currently, only the models with static shapes are supported on NPU.
-* If the path to the model file includes non-Unicode symbols, such as in Chinese,
-  the model cannot be used for inference on NPU. It will return an error.
-* Running the Alexnet model with NPU may result in a drop in accuracy.
```

Comment:
This is not the only model that could show lower accuracy unfortunately. I see no good reason to point this one out here.

Reply:
Alexnet accuracy has been fixed, so it definitely requires this note to be deleted. There are the other models, but the list of accurate models will be published (if it will be) separately.